src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -449,20 +449,13 @@ let main argv =
                 exit 1
             | [], Some fscArgs -> runFscArgs client fscArgs ignoreFiles severityMapping |> Async.RunSynchronously
             | projects, None ->
-                let runProj (proj: string) =
-                    async {
-                        let project =
-                            if Path.IsPathRooted proj then
-                                proj
-                            else
-                                Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, proj))
-
-                        let! results = runProject client toolsPath project ignoreFiles severityMapping
-                        return results
-                    }
+                for projPath in projects do
+                    if not (File.Exists(projPath)) then
+                        printError $"Invalid `--project` argument. File does not exist: '{projPath}'"
+                        exit 1
 
                 projects
-                |> List.map runProj
+                |> List.map (fun projPath -> runProject client toolsPath projPath ignoreFiles severityMapping)
```

Comment:
Did you test this with an absolute path?
What happens in `runProject` when you do?

Reply:
I did. As we use `Path.Combine` in `runProject`, relative and absolute paths still work.
From the docs of `Path.Combine`:
`The combined paths. If one of the specified paths is a zero-length string, this method returns the other path. If path2 contains an absolute path, this method returns path2.`

Reply:
Alright!